nss-day-cohort-46/reactive-nutshell-talisman-of-havocing
# Nc adding event component

# Description
The event form will save a new event to the api
## Type of change


# Testing Instructions
git add all on your branch
git commit on your branch
ttb nc-addingEventComponent

Open the site in your browser. Fill out the event form that renders after you log in. Hit save. See that event in your database.json file.

# Checklist:
